burotel/indico_burotel/controllers.py

```diff
@@ -7,15 +7,25 @@
 
 from __future__ import unicode_literals
 
+from babel.dates import format_datetime
```

Comment:
shouldn't we use the one from the indico core since they already handle using e.g. the session language? (even though that one requires manually converting back to unicode since those utils still encode to utf8...)

Reply:
Indeed. Fixed.